src/components/GridBoard.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from "react";
+import GridHex from "./GridHex";
+import Layout from "./Layout";
+import Piece from "@components/Piece";
+import { Point } from "@datatypes/Piece";
+
+const gridHexCoordinates: Point[] = [];
+
+for (let row = 0; row < 13; row++) {
+  for (let col = 0; col < 16; col++) {
+    gridHexCoordinates.push({ row, col });
+  }
+}
+const samplePieceCoordinates: Point[] = [
+  { row: 1, col: 0 },
+  { row: 5, col: 8 },
+  { row: 3, col: 4 },
+  { row: 3, col: 14 },
+  { row: 3, col: 5 },
+  { row: 4, col: 8 },
+];
+
+const GridBoard = () => {
+  const [pieceCoordinates, setPieceCoordinates] = useState(
+    samplePieceCoordinates,
+  );
+
+  const [clickedPiece, setClickedPiece] = useState<Point | null>(null);
+
+  const handlePieceClick = ({ row, col }: Point) => {
+    setClickedPiece({ row, col });
+  };
+
+  const handldGridHexClick = ({ row, col }: Point) => {
```

Comment:
typo haha

Reply:
oop LMAO i had to glance over it thrice to see what's wrong